src/js/actions/search/commands.js

```diff
@@ -161,7 +162,85 @@ define(function (require, exports) {
      * @param {string} id ID of menu command
     */
     var _confirmMenuCommand = function (id) {
-        menuActions._playMenuCommand.call(this, id);
+        menuActions.playMenuCommand.call(this, id);
+    };
+
+    /**
+     * Translates the modifiers and key to a string for the UI
+     *
+     * @private
+     * @param {object} fullShortcut
+     * @return {string}
+     */
+    var _getShortcutString = function (fullShortcut) {
+        var modifierBits = fullShortcut.modifiers,
+            key = fullShortcut.key,
+            modifierStrings = strings.SEARCH.MODIFIERS,
+            shortcut = "";
+
+        if (modifierBits.command) {
+            shortcut += "\u2318";
+        }
+
+        if (modifierBits.control) {
+            shortcut += system.isMac ? "^" : modifierStrings.CONTROL;
+        }
+
+        if (modifierBits.alt) {
+            shortcut += system.isMac ? "\u2325" : modifierStrings.ALT;
+        }
+
+        if (modifierBits.shift) {
+            shortcut += "\u21E7";
+        }
+
+        if (key) {
+            shortcut += key.toString().toUpperCase();
+        }
+
+        return " " + shortcut + "\u00a0\u00a0\u00a0\u00a0";
```

Comment:
@shaoshing Alison's original code had 4 spaces, so I am assuming she talked about it with @placegraphichere and they agreed upon this for the way the shortcut renders in the Search Bar UI.